x/docs/md/cli/linux.md

```diff
@@ -1,7 +1,7 @@
 
 ## Linux
 
-View the CLI releases to [choose and download the appropriate package for your operating system](https://github.com/exercism/cli/releases/tag/v2.3.0).
+View the CLI releases to [choose and download the appropriate package for your operating system](https://github.com/exercism/cli/releases).
```

Comment:
Let's use `https://github.com/exercism/cli/releases/latest` which will automatically redirect to the most recent one.